draft-jholland-quic-multicast.md

```diff
@@ -121,17 +121,21 @@ Incoming packets received on the network path associated with a channel use the
 A client with a matching joined channel always has at least one connection associated with the channel.
 If a client has no matching joined channel, the packet is discarded.
 
-Each channel has an independent packet number space. To enable clients to detect lost packets, packet numbers in channels MUST be continuous.
-Since the network path for a channel is unidirectional and uses a different packet number space than the unicast part of the connection, packets associated with a channel are acknowledged with MC_ACK frames {{channel-ack-frame}} instead of ACK frames.
+## Channel using Multipath QUIC
+
+From the point of view of the client, each Multicast QUIC channel is handled as a second path from the server. A client keeps its unicast connection with the server open during all the transmission. Additionally, it negotiates a second path with the server where it will receive multicast content. All mechanisms, except those listed below, follow {{I-D.draft-ietf-quic-multipath}}.
+
+The first major change concerns the encryption of the multicast path. To keep the unicast path between the client and the server secure, the multicast path must use a different cryptographic context that is common between all clients. As such, each path is encrypted differently, differing from the current version of {{I-D.draft-ietf-quic-multipath}}. NB: there are discussions within the Multipath QUIC draft to integrate this feature to strengthen the security of MPQUIC. Secondly, a client MUST NOT send any data on the multicast path with the server.
```

Comment:
To me it's not quite clear if there is a need for a full cryptographic context at all for the channels or if the algorithm carried in announce frames plus the key from the key frames is enough. (Though I guess that is somewhat nitbitting)

Reply:
Indeed we have to specify what we mean by "a cryptographic context". As long as each data sent on the multicast channel is encrypted and decrypted with a different key than the unicast key (where I used the word "context"), it should be secure for the communication